src/Intent.yaml

```diff
@@ -23,6 +23,16 @@ Intent:
       description: The contexts the intent accepts. This will
         typically be a set of namespaced context types, e.g. 
         "org.symphony.contact"
+    dataReturnType: 
+      type: string
+      description: If there is a data payload associated with the intent resolution, specifies the type.  
+      This value can be either a URL to the type definition, or an alias for the type.  
+      If this value is set, the desktop agent can manage the resolution of the intent accordingly.
```

Comment:
I am not a fan of this. The difference between a data-driven intent and fire-and-forget intent is then implicit based on whether this property is present or not. Lots of opportunity for misunderstanding.

I would like it to be more explicit: An "intentType" of some kind that indicates the difference, or at the top level a different object for "intent".